src/api/chat/chats.ts

```diff
@@ -0,0 +1,45 @@
+import { HTTPTransport } from '../api';
+import { ChatType, UsersChat } from "@/pages/chats/types";
+
+export class chatsAPI {
+    request: HTTPTransport;
+    baseUrl: string;
+
+    constructor() {
+        this.request = new HTTPTransport();
+        this.baseUrl = 'https://ya-praktikum.tech/api/v2';
+    }
+
+    async getChats(): Promise<ChatType[]> {
+        return await this.request.get<ChatType[]>(`${this.baseUrl}/chats`, {});
```

Comment:
Можно сделать options необязательным параметром, чтобы не передавать пустой объект

Reply:
Здесь необязательно писать async/await 
```suggestion
        return this.request.get<ChatType[]>(`${this.baseUrl}/chats`, {});
```
вернет промис